src/components/nav-bar/navlink.tsx

```diff
@@ -0,0 +1,44 @@
+import { Block } from 'baseui/block';
+import { StyledLink } from 'baseui/link';
+import { Link } from 'gatsby';
+import React from 'react';
+
+type NavLinkProps = {
+  navigateTo: string;
+  text: string;
+  textColor: string;
+};
+
+const NavLink: React.FC<NavLinkProps> = ({ navigateTo, text, textColor }) => {
+  return (
+    <Block
+      overrides={{
+        Block: {
+          style: ({ $theme }) => ({
+            fontWeight: 400,
+            fontSize: $theme.sizing.scale550,
+          }),
+        },
+      }}
+    >
+      <Link to={navigateTo} style={{ textDecoration: 'none' }}>
+        <StyledLink
+          $style={({ $theme }) => ({
+            color: `${textColor} !important`,
```

Comment:
Using `!important` is not a good practice. See: https://uxengineer.com/css-specificity-avoid-important-css/ (there are other online literature around it).

We need to make sure we understand the css specificity correctly and apply css accordingly.